src/explore-education-statistics-admin/src/pages/methodology/external-methodology/ExternalMethodologyPage.tsx

```diff
@@ -26,7 +25,12 @@ const ExternalMethodologyPage = ({
   const { value: model, isLoading } = useAsyncHandledRetry<Model>(async () => {
     const [publication, externalMethodology] = await Promise.all([
       publicationService.getPublication(publicationId),
-      publicationService.getExternalMethodology(publicationId),
+      publicationService.getExternalMethodology(publicationId).catch(err => {
+        if (err.response.status !== 404) {
+          throw err;
+        }
+        return undefined;
+      }),
```

Comment:
Let's do this in the service method so we don't need to do this repeatedly all over the place. Would either return null (for a 404) or re-throw the error (as you have). 

We'll also need to change all the test mocks to use `mockResolvedValue` again instead, sorry!

```ts
publicationService.getExternalMethodology.mockResolvedValue(null);
```